incubator/kube-aws-ingress-controller/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+appVersion: 0.1
+name: kube-aws-ingress-controller
```

Comment:
Does it makes sense to change the name to kube-ingress-aws-controller, because the GH repo of the project has this name?

Reply:
Done :) Should be correct now